packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -139,9 +147,9 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
      * @param error - optional error object to log
      */
      protected sendTelemetryEventCore(
-        event: ITelemetryGenericEvent & { category: TelemetryEventCategory; },
+        event: ITelemetryGenericEventExt & { category: TelemetryEventCategory; },
         error?: any) {
-        const newEvent = { ...event };
+        const newEvent = convertToBaseEvent(event) as ITelemetryBaseEventExt;
```

Comment:
I'm wondering if I'm missing something, but this should be fine to just be the old `ITelemetryBaseEvent`
```suggestion
        const newEvent = convertToBaseEvent(event);
```

Reply:
committed suggested change